collect/stressRelief.go

```diff
@@ -99,6 +135,26 @@ func (s *StressRelief) Start() error {
 		{Numerator: "memory_heap_allocation", Denominator: "MEMORY_MAX_ALLOC", Algorithm: "sigmoid", Reason: "MaxAlloc"},
 	}
 
+	var hostID string
```

Comment:
I just had an idea -- since peers have to generate an identifier anyway, is there any value in adding an interface to Peers to GetCurrentID()? It wouldn't be the hostname, but the IP address. It would correlate with the peer traffic, but no longer would match the k8s name. 

Do you have an opinion?

Reply:
I like the idea of having a uniformed way within Refinery to identify a n instance. Since we have stress relief metrics that contains hostname already, I think it's fine to use IP address as the identifier